server/src/main/java/io/github/tonygermano/userprivacy/server/UserPrivacyServicePlugin.java

```diff
@@ -18,16 +18,17 @@
 
 package io.github.tonygermano.userprivacy.server;
 
-import com.kaurpalang.mirth.annotationsplugin.annotation.ServerClass;
+import com.kaurpalang.mirth.annotationsplugin.annotation.MirthServerClass;
 import com.mirth.connect.model.ExtensionPermission;
 import com.mirth.connect.plugins.ServicePlugin;
 import com.mirth.connect.server.controllers.ControllerFactory;
-import java.util.Map;
-import java.util.Properties;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-@ServerClass
+import java.util.Map;
+import java.util.Properties;
```

Comment:
why move these down?

Reply:
I used `Ctrl-Alt-O` to clear unused imports and those two were likely moved by IntelliJ. Not an intentional change :)